instrumentation/spring/spring-boot-actuator-autoconfigure-2.0/javaagent/build.gradle.kts

```diff
@@ -22,6 +22,16 @@ dependencies {
   latestDepTestLibrary("ch.qos.logback:logback-classic:+")
 }
 
+testing {
+  suites {
+    val testPrometheus by registering(JvmTestSuite::class) {
+      dependencies {
+        runtimeOnly("io.micrometer:micrometer-registry-prometheus:1.14.3")
+      }
+    }
+  }
+}
```

Comment:
since there are no source for this suite it won't run any tests as far as I can tell

Reply:
OK - I had to test this by adding ` id("com.adarshr.test-logger") version "4.0.0"` to see that noting gets executed...

used the setup for stable semconv now